app/views/angular_template/test.html

```diff
@@ -0,0 +1,2 @@
+<!-- Please do not remove, will break tests :) -->
+It works!
```

Comment:
> That'd work but ..chicken and egg :). I'm expecting @zeari to use it once it's merged but until it is and his code is, we have no real files to test. So I can either remove it for now, or leave it as it is and change to load real files once we have some.
> 
> EDIT.. and until we do have real files, I'd rather not remove it since it doubles as an usage example :)

@himdel , @zeari perhaps you can discuss together if one of the tile/status cards of patternfly that was the base of this discussion/pr, can be added as part of this PR as a real (and needed) example. (and then used by the dashboard PR). This way this will add a needed and real code and not just placeholders (and will test whether the solution is suffice and working)